.github/workflows/build.yml

```diff
@@ -29,6 +29,10 @@ jobs:
           role-skip-session-tagging: true
           role-duration-seconds: 900
 
+      - name: Upload Env File to S3
+        run: |
+          aws s3 cp cd/application-deployment/${{ inputs.environment }}/${{ inputs.environment }}.env s3://vanotify-environment-variables-${{ inputs.environment }}/
```

Comment:
Can we make these filenames unique? I'm imagining a scenario where multiple people are trying to deploy to dev and they overwrite each other's files. When I did this in the past, we used the commit hash as the object name.

Reply:
Spoke to @EvanParish and we decided that naming these things with a sha would actually add more work and complexity for almost no additional value. Resolving this comment.